### PR TITLE
Script minification fix (after Node version upgrade)

### PR DESCRIPTION
**Pull request by saedwards**
### What is the context of this PR?
Describe what you have changed and why, link to other PRs or Issues as appropriate.

### How to review 
Describe the steps required to test the changes (include screenshots if appropriate).

### CODE REVIEW

**Comment by armstrongb** [comment]
Looks good to me. @nealedj can you take a quick look too?

File: gulpfile.babel.js [hunk 1]
```diff
@@ -213,116 +213,11 @@ const bundleScripts = watch => {
 
   return gulp.src(['./assets/js/polyfills.js', './assets/js/components.js'])
     .pipe(bundler())
-    .pipe(gulp.dest('./public/assets/scripts'));
-
-
-  /*return browserify(['./assets/js/polyfills.js', './assets/js/components.js'])
```
Comment (jonnyshaw89) [inline gulpfile.babel.js]: If we don't need this code anymore then we should delete it.

Comment (armstrongb) [reply gulpfile.babel.js]: @saedwards can you resolve this one then I can get this into the v1.4.2 release. Thanks.

**Comment by armstrongb** [comment]
@jonnyshaw89 can you re-review when you have a few minutes?

**Pull request merged**